src/fetch.js

```diff
@@ -1,12 +1,13 @@
 import axios from 'axios'
 import { isObject, startsWith, forEach } from 'lodash'
+import pluralize from 'pluralize'
 
 module.exports = async ({ apiURL, contentType, jwtToken }) => {
   console.time('Fetch Strapi data')
-  console.log(`Starting to fetch data from Strapi (${contentType})`)
+  console.log(`Starting to fetch data from Strapi (${pluralize(contentType)})`)
 
   // Define API endpoint.
-  const apiEndpoint = `${apiURL}/${contentType}`
+  const apiEndpoint = `${apiURL}/${pluralize(contentType)}`
```

Comment:
@aDEVE your changes lead to an error on my side.  More concretely this line in  causes a `404` Error during the `gatsby develop` step.

I am using Strapi 3.0.0-alpha.14 and my endpoints are not pluralized 

e.g. the endpoint is `/product` but the pluralization changes it in `/products`. Could you give me some insights why you added the pluralization. I am pretty sure you had a good reason to do so (and it seems to work for you)